src/climate_learn/metrics/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Local application
+from .metrics import *
```

Comment:
Yes, this is to avoid cluttering the namespace. My intention is for users to load the class and functional definitions separately, like in PyTorch. For example,
```python
import climate_learn.metrics as metrics
import climate_learn.metrics.functional as CLF
```
which mirrors
```python
import torch.nn as nn
import torch.nn.functional as F
```

Reply:
I see. That makes sense.